static/app/components/group/suggestedOwners/ownershipRules.tsx

```diff
@@ -78,3 +144,51 @@ const HelpfulBody = styled('div')`
   padding: ${space(1)};
   text-align: center;
 `;
+
+const Container = styled(Panel)`
+  width: 100%;
+  height: 100%;
```

Comment:
Do you need this height or width prop?

Reply:
@EvanPurkhiser nah its legacy styles from before using Panel